cmd/nerdctl/network_rm.go

```diff
@@ -93,14 +93,17 @@ func networkRmAction(cmd *cobra.Command, args []string) error {
 
 	for _, name := range args {
 		if name == "host" || name == "none" {
-			return fmt.Errorf("pseudo network %q cannot be removed", name)
+			fmt.Fprintf(cmd.OutOrStdout(), "Error: pseudo network %q cannot be removed\n", name)
+			continue
 		}
 
 		n, err := walker.Walk(cmd.Context(), name)
 		if err != nil {
-			return err
+			fmt.Fprintf(cmd.OutOrStdout(), "Error: %s\n", err.Error())
+			continue
 		} else if n == 0 {
-			return fmt.Errorf("no such network %s", name)
+			fmt.Fprintf(cmd.OutOrStdout(), "Error: No such network: %s\n", name)
```

Comment:
please use `logrus.Error(err)`  or `logrus.WithError(err).Errorf("failed to ... %s", var)` to set the Error level for these logs